sandbox/elephantsdream.ad.vtt.example

```diff
@@ -0,0 +1,278 @@
+WEBVTT

```

Comment:
Actually, we should substitute [this file](http://html5videoguide.net/presentations/LCA_MM_Miniconf2013/elephant.audiodescriptions.vtt), which contains the attribution and license for this file from @silviapfeiffer.